wiki/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021, Mycroft AI Inc.
```

Comment:
This feels weird to me. The Skill itself is in the top-level dir. The wiki directory contains a wrapper for Wikipedia.

I'm a fan of consistency however in this instance the consistency seems to make it less intuitive.

Reply:
imagine if every package was named `package`, thats what the `skill` folder feels like to me, I'm not a fan of that new pattern

Reply:
This is a forward thinking request.  The skill itself will be moved into this package as soon as Core supports it and there will be no `__init__.py` file in the base skill directory.  I chose the name `skill` to be generic enough that all skills could have a Python package of the same name.  Thus, making the skill loader code simpler and easier to maintain as it will always look in the same place for the skill class (just as it does today).  Traversing all the skill's files looking for the class because the package is named differently in each skill is bad design, IMO.

@JarbasAl if you have a better name, I am all ears.  Sorry you aren't a fan, but unless a better alternative is presented, this is how skills will be structured going forward.

Reply:
Going to leave this for now, and we can revisit with the Skill restructuring discussions.